src/app/features/admin/components/carriage-form/carriage-form.component.html

```diff
@@ -0,0 +1,34 @@
+<form class="tui-form__row" [formGroup]="carriageForm">
+  <div class="tui-form__row">
+    <tui-input formControlName="name">
+      Name
+      <input tuiTextfieldLegacy />
+    </tui-input>
+    <tui-error formControlName="name" [error]="[] | tuiFieldError | async" />
+  </div>
+  <div class="tui-form__row">
+    <tui-input-number [min]="min" class="input" formControlName="rows"> Rows </tui-input-number>
+    <tui-error formControlName="rows" [error]="[] | tuiFieldError | async" />
+  </div>
+  <div class="tui-form__row">
+    <tui-input-number [min]="min" class="input" formControlName="leftSeats"> LeftSeats </tui-input-number>
+    <tui-error formControlName="leftSeats" [error]="[] | tuiFieldError | async" />
+  </div>
+  <div class="tui-form__row">
+    <tui-input-number [min]="min" class="input" formControlName="rightSeats"> RightSeats </tui-input-number>
+    <tui-error formControlName="rightSeats" [error]="[] | tuiFieldError | async" />
+  </div>
```

Comment:
You can use ng-template with ng-container and ngTemplateOutlet to get rid of code duplication https://www.youtube.com/watch?v=vfPVdJ2oQlM